src/routers/handlers/enter_your_details/index.ts

```diff
@@ -0,0 +1,96 @@
+import { BaseViewData, GenericHandler, Redirect, ViewModel } from "./../generic";
+import { Request, Response } from "express";
+import { logger } from "../../../utils/logger";
+import { type Address } from "private-api-sdk-node/src/services/presenter-account/types";
+import { PrefixedUrls, Countries } from "../../../constants";
+import { setPresenterAccountDetails, getPresenterAccountDetails, generateUserDetails } from "./../../../utils/session";
+import { FieldValidationError, Result, ValidationError, validationResult } from "express-validator";
+
+interface CountryType {
+    value: string,
+    text: string,
+    selected?: boolean
+}
+
+interface ErrorManifest {
+    [key: string]: any
+}
```

Comment:
This type isn't specific to this handler so it should be defined in the error manifest package (`src/utils/error_manifests/default.ts`).
Additionally, the value isn't typed correctly. It should be an object with an `inline` and `sumary` fields which both hold strings.